code/ReportAdmin.php

```diff
@@ -104,13 +111,18 @@ public function canView($member = null)
      */
     public function Reports()
     {
-        $output = new ArrayList();
+        $output = ArrayList::create();
         foreach (Report::get_reports() as $report) {
             if ($report->canView()) {
                 $output->push($report);
             }
         }
-        return $output;
+
+        return $output
+            // Provide a good default for the reports order, otherwise this will be using sort by namespace
+            ->sort('Title', 'ASC')
+            // Provide a good default for the data class, otherwise this will use the class of the first item in the list
+            ->setDataClass(Report::class);
```

Comment:
```suggestion
            ->sort('Title', 'ASC')
            ->setDataClass(Report::class);
```
I think those are pretty self-explanatory.